### DISCUSSION TITLE
Printing test output to the regular console instead of repl

### DISCUSSION THREAD

**Discussion by asmodeus812** [mfussenegger/nvim-jdtls#438] [General]
Hey, i was just playing around with the test dap runners and was wondering if it is possible to instead redirect the output to the regular dap console instead other repl, maybe we could provide a configuration when invoking the api method in case you prefer the repl to be the default behaviour. Thanks !

**Comment by gbroques** [comment]
@asmodeus812 This may be possible.

You could try what @mfussenegger suggested in the following post:
https://github.com/mfussenegger/nvim-jdtls/issues/31#issuecomment-800601661

```
test_nearest_method({ config = { console = 'console' }})
```

**See Also:** https://github.com/mfussenegger/nvim-dap/blob/master/doc/dap.txt#L555-L558
